api/v1alpha/api/openapi.yaml

```diff
@@ -61866,14 +62148,6 @@ components:
           - ASIA_SOUTHEAST_2
           type: string
           x-xgen-go-transform: merge-oneOf
-        analyticsAutoScaling:
```

Comment:
Those are child fields (AWS region config) that were removed. 
Do we know why? Is that ok?